app/scraper.py

```diff
@@ -13,6 +14,8 @@
 
 DB = Database()
 
```

Comment:
If not needed can we remove the additional DB = Database() here?

Reply:
Another good catch, when I was doing git rebase it seems I added that.